src/icrc2-minter/src/state.rs

```diff
@@ -87,9 +86,8 @@ impl Default for Settings {
             owner: Principal::anonymous(),
             evm_principal: Principal::anonymous(),
             signing_strategy: SigningStrategy::Local {
-                private_key: [1u8; 32],
+                private_key: [218u8; 32],
```

Comment:
What is this for?

Reply:
To prevent accidental coincidence of private keys of different test cansiters.